maquetado/login.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body class="body">
+    <form class="login-container">
+      <img class="wolox-image-login" src="image/LogoWolox.png" alt="Wolox" />
+      <label for="email" class="label">Email</label>
+      <input class="input" type="text" id="email" />
```

Comment:
Next time you can use `<input type="email" ...`
Here you can the all the types an input can have: [https://www.w3schools.com/html/html_form_input_types.asp](https://www.w3schools.com/html/html_form_input_types.asp)